src/view/treeNodes/pullRequestNode.ts

```diff
@@ -54,11 +54,6 @@ export class PRNode extends TreeNode implements vscode.CommentingRangeProvider2
 		super(parent);
 		this.registerSinceReviewChange();
 		this.registerConfigurationChange();
-		this._register(this.pullRequestModel.onDidChange((e) => {
-			if (e.title || e.state) {
-				this.refresh(this);
-			}
-		}));
```

Comment:
We listen to this event and refresh based on it in another place. Doing so here as well results in double the refreshes.